src/Chapter2.hs

```diff
@@ -780,13 +812,15 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
 -- TODO: type ;)
-listElementsLessThan x l = filter (< x) l
+listElementsLessThan :: Int -> [Int] -> [Int]
+listElementsLessThan = (\x l -> filter (< x) l)
 
 -- Can you eta-reduce this one???
-pairMul xs ys = zipWith (*) xs ys
+pairMul :: [Int] -> [Int] -> [Int]
+pairMul = (\x y -> zipWith (*) x y)
```

Comment:
This is actually not an eta-reduction, because you still have variable names :slightly_smiling_face: 

See the example with `divideTenBy`: in there you remove the `x` variable completely from both sides.